README.md

```diff
@@ -148,6 +148,23 @@ A short definition is "a sequence of elements from a source that supports aggreg
   _DataSource object_ provides connection to _th2-rpt-provider_ or read csv files from cradle-viewer.
 - **Aggregate operations**: 
   Common operations such as filter, map, find and so on. 
+- **Data caching**:
+  The _Data object_ provides the ability to use the cache. 
+  The cache works for each _Data object_, that is, you choose which _Data object_ you want to save. 
+  The _Data object_ cache is saved after the first iteration, but the iteration source may be different.
+  If you don't use the cache, your source will be the data source you have in the _Data Object_. 
+  But if you use the cache, your source can be the data source, the parent cache, or own cache:
+  * The data source: 
+  If the "Data Object" doesn't have a parent cache and its cache.
+  * The parent cache: 
+  If the "Data Object" has a parent cache. 
+  It doesn't matter what position the parent cache has in inheritance. 
+  "Data Object" understands whose cache it is and applies the part of the workflow that was not applied.
```

Comment:
"Data Object" understands whose cache it is and executes the part of the workflow that was not executed.